.github/ISSUE_TEMPLATE/tech_debt.md

```diff
@@ -0,0 +1,29 @@
+### Description
+A clear and concise description of what the tech debt is and the reason of being created
+
+### Impact
+Description of the current or future impact of this tech debt.
+
+### Risk
```

Comment:
Impact and Risk seem like kind of the same thing to me? In most cases, I'd think the impact is that we'll get rid of whatever the risk is